CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Added feature to set the environment variable `CI_SKIP=true` when using `continue-after-seconds` so that only a subset of future steps can be skipped.
```

Comment:
```suggestion
* Added feature to set the output `force_continued=true` when using `continue-after-seconds` so that only a subset of future steps can be skipped.
```